docs/root/configuration/http/http_filters/on_demand_updates_filter.rst

```diff
@@ -15,10 +16,20 @@ The on-demand update filter can also be used to request a *Route Configuration*
 loaded on demand in the :ref:`Scoped RouteConfiguration <envoy_v3_api_msg_config.route.v3.ScopedRouteConfiguration>`.
 The contents of the HTTP header is used to find the scope and create the on-demand request.
 
+The on-demand update filter can also be used to request a cluster when the target cluster name is only known after receiving
+a request and the cluster is missing - this is a scenario that could happen with the cluster_header route action. For an on demand
```

Comment:
Can you use literal formatting or links for things like `cluster_header`, `ocds_config`, etc?

Reply:
Done.